MekHQ/src/mekhq/gui/StratconTab.java

```diff
@@ -56,6 +60,8 @@
     private JScrollPane expandedObjectivePanel;
     private boolean objectivesCollapsed = true;
 
+    CampaignManagementDialog cmd;
```

Comment:
I can see no good reason to have this always initialized and not just recreate it as required.

Reply:
On the other hand, I don't see a reason to keep creating/destroying a bunch of buttons and whatnot.

Reply:
That's how we handle it in MekHQ, and... permanent memory vs clearable memory regularly matters when using a weaker computer.